docs/source/quick-start.rst

```diff
@@ -51,3 +51,9 @@ Connection pooling
 
 .. literalinclude:: examples/pooling.py
    :language: python
+
+Connection pooling with recycled channels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Please add some context here. Why it's useful? When pool might be used without this example? Why `Connect` class is not a replacement for `NonRestoringRobustConnection`? etc.

This will be as useful as the code itself.

Reply:
Added